e2e/setup/global-setup.ts

```
@@ -0,0 +1,238 @@
+  const { instance, passphrase } = user
+  const loginPageRes = await fetch(`http://${instance}:80/auth/login`)
+  const html = await loginPageRes.text()
+
+  const csrfToken = matchOrThrow(
+    html,
+    /name="csrf_token"\s+value="([^"]+)"/,
+    'CSRF token',
+    instance
+  )
+  const iterations = parseInt(
+    matchOrThrow(html, /data-iterations="(\d+)"/, 'PBKDF2 iterations', instance),
```

ESLint

[error] Replace `html,·/data-iterations="(\d+)"/,·'PBKDF2·iterations',·instance` with `⏎······html,⏎······/data-iterations="(\d+)"/,⏎······'PBKDF2·iterations',⏎······instance⏎····`

(prettier/prettier)